lib/plottr_components/src/components/dialogs/HierarchyLevel.js

```diff
@@ -66,7 +67,7 @@ const HierarchyLevelConnector = (connector) => {
           setValue={setValue('textColor')}
```

Comment:
I'm not sure that we updated this change handler to support the new schema?
You'll probably need to write a new version of `setValue` that sets the prop in the appropriate theme and sets it on the root object for backwards compatibility.  e.g.

```javascript
    const setThemedValue = (name) => (newValue) => {
      editHierarchyLevel({
        level: level,
        [name]: newValue,
        [theme]: {
          textColor,
          borderColor,
          [name]: newValue,
        }
      })
    }
```

Note that I included the old values for the `[theme]` key and then override them so that the state is merged correctly in the reducer.